### DISCUSSION TITLE
Nox version update

### DISCUSSION THREAD

**Discussion by geniusdragon** [tonalOrchestrator/mff_auto#32] [Q&A]
Hi! Thank you for continuing the work on this code - I was hoping to do something similar myself, but I'm just not skilled enough yet! I was wondering - before it was archived, the original developer was testing using ADB for a direct connection to nox to bypass version issues. Was this something you'd be able to look at?

**Comment by tonalOrchestrator** [comment]
Hi @geniusdragon I can try to look into this.   I thought that this was mainly for Blue Stacks though but I can follow the original thread.   As I thought that bluestacks couldn't restart the game, I have personal interest in updating this application but unfortunately I don't think that ADB would help with the updated nox version.   I installed the latest Nox version to see if I could get things to play nice but there were still the same issues on the window handles that I just can't figure out.    I don't have a ton of time or experience with the framework entirely, but I can tell you there are a lot of customizations to make this thing work.
